Added class for tests with network creation

alexgrom9393 opened the pull request:
Added TestDeployEnvInNetwork for test where we should add network, subnet,
router and some interfaces for them. It contains fixture for test
preparation and tearDown function for cleanup.
Modified function for routers deletion.

Commit:
Added class for tests with network creation

Added TestDeployEnvInNetwork for test where we should add network, subnet,
router and some interfaces for them. It contains fixture for test
preparation and tearDown function for cleanup.
Modified function for routers deletion.

gdyuldin commented on mos_tests/murano/test_sanity.py:
```diff
@@ -140,3 +141,65 @@ def test_import_package_from_repo(self):
+            subnet=subnet)
+
+    @classmethod
+    def tearDownClass(cls):
```
I believe, you don't need to combining fixtures and setup/teardown approach.
Let's move cleanup to `prepare` fixture, and rename in (perhaps to `networks`)

Commit:
Applying comments. Added 3 murano tests.

1. Applied comments
2. Added test on deployment Murano environment in particular network.
3. Added test on deployment Murano environment in particular network when
network doesn't have empty IP address.
4. Added test on verification package_size_limit.
